specification/configuration.md

```diff
@@ -222,7 +222,8 @@ are required.
   - Distribution MUST NOT list Zipkin exporter as supported (not supported by Smart Agent)
 - `OTEL_TRACES_EXPORTER`
   - Non-RUM distribution MUST default to `otlp` over gRPC with an endpoint of `localhost:4317`
-  - Non-RUM distribution MUST offer `jaeger-thrift-splunk` that defaults to `http://127.0.0.1:9080/v1/trace`
+  - Non-RUM distribution MAY offer `jaeger-thrift-splunk` that defaults to `http://127.0.0.1:9080/v1/trace`. **`jaeger-thrift-splunk` is deprecated. Distributions SHOULD provide a deprecation warning.**
```

Comment:
Is there any reason why not using "MUST" instead of "SHOULD" regarding the deprecation warning?

Also I think we could try to reuse the wording from the scenario when `OTEL_SERVICE_NAME` is empry. Proposal:

```md
  - Non-RUM distribution MAY offer `jaeger-thrift-splunk` that defaults to `http://127.0.0.1:9080/v1/trace`.
     **`jaeger-thrift-splunk` is deprecated.
     If the user uses `jaeger-thrift-splunk` the distribution MUST log a
     warning. The warning message MUST clearly describe it and propose an alternative. E.g.

     ` ` ` (remove spaces)
     jaeger-thrift-splunk trace exporter is deprecated and may be removed in a future release.
     We suggest that you use the default otlp exporter instead. You can set the SPLUNK_REALM
     environmental variable to send spans directly to Splunk Observabilty Cloud. 
     ` ` ` (remove spaces)
```

Reply:
Changed it to MUST, wasn't completely sure here. Check the wording now